code/modules/antagonists/heretic/items/heretic_necks.dm

```diff
@@ -9,6 +9,106 @@
+
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/equipped(mob/living/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_NECK)
+		return
+
+	var/team_color = COLOR_ADMIN_PINK
+	if(IS_CULTIST(user))
+		RegisterSignal(user, COMSIG_LIVING_CULT_EMPOWER, PROC_REF(buff_empower))
+		team_color = COLOR_CULT_RED
+	if(IS_HERETIC_OR_MONSTER(user))
+		RegisterSignal(user, COMSIG_ACTION_START_COOLDOWN, PROC_REF(halve_cooldowns))
+		team_color = COLOR_GREEN
+	else
+		team_color = pick(COLOR_CULT_RED, COLOR_GREEN)
+
+	user.add_traits(list(TRAIT_MANSUS_TOUCHED, TRAIT_BLOODY_MESS), REF(src))
+	to_chat(user, span_alert("Your heart takes on a strange yet soothing irregular rhythm, and your blood feels significantly less viscous than it used to be. You're not sure if that's a good thing."))
+	component = user.AddComponent( \
+		/datum/component/aura_healing, \
+		range = 3, \
+		brute_heal = 0.1, \
+		burn_heal = 0.1, \
+		blood_heal = 2, \
+		suffocation_heal = 5, \
+		simple_heal = 0.6, \
+		requires_visibility = FALSE, \
+		limit_to_trait = TRAIT_MANSUS_TOUCHED, \
+		healing_color = team_color, \
+		)
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/dropped(mob/living/user)
+	. = ..()
+
+	if(!istype(user))
+		return
+
+	if(HAS_TRAIT_FROM(user, TRAIT_MANSUS_TOUCHED, REF(src)))
+		to_chat(user, span_notice("Your heart and blood return to their regular old rhythm and flow."))
+
+	UnregisterSignal(user, list(COMSIG_LIVING_CULT_EMPOWER, COMSIG_ACTION_START_COOLDOWN))
+	if(component)
+		QDEL_NULL(component)
+	user.remove_traits(list(TRAIT_MANSUS_TOUCHED, TRAIT_BLOODY_MESS), REF(src))
+
+	if(!IS_CULTIST(user))
+		return
+	// Remove the fifth spell slot, if any.
+	var/datum/action/innate/cult/blood_magic/magic_holder = locate() in user.actions
+	if(!magic_holder)
+		CRASH("cultist with no blood magic holder?")
+	if(length(magic_holder.spells) > 4)
+		QDEL_NULL(magic_holder.spells[5])
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/proc/buff_empower(mob/user, signal_return_list)
+	SIGNAL_HANDLER
+
+	signal_return_list["limit_data"] += 1
+	signal_return_list["speed_data"] *= 0.5
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/proc/halve_cooldowns(mob/user, signal_return_list)
+	SIGNAL_HANDLER
+
+	signal_return_list["cd_data"] = signal_return_list["override_data"] ? signal_return_list["override_data"] * 0.5 : signal_return_list["cd_data"] * 0.5
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/attack_self(mob/living/user, modifiers)
+	. = ..()
+	to_chat(user, span_danger("You start tightly squeezing [src]..."))
+	if(!do_after(user, 1.25 SECONDS))
+		return
+	to_chat(user, span_danger("[src] explodes into a shower of gore and blood, drenching your arm. You can feel the blood seeping into your skin. You inmediately feel better, but soon, the feeling turns hollow as your veins itch."))
+	new /obj/effect/gibspawner/generic(get_turf(src))
+	var/heal_amt = user.adjustBruteLoss(-50)
+	user.adjustFireLoss( -(50 - heal_amt) ) // no double dipping
```

Comment:
This might need to be verified because `adjustBruteLoss` returns a negative value, 
meaning this might heal `-(50 - -50) -> `-100`

Reply:
Didnt notice that in testing but went ahead and made it abs just in case